README.rst

```diff
@@ -82,17 +82,17 @@ The following table describes each of the arguments:
  n_concurrent=<n_concurrent>   Specifies the number of concurrent requests to submit. By default, 4 concurrent requests are submitted. Set ``<n_concurrent>`` to match the number of cores in the prediction API endpoint.
  create_api_token              Requests a new API token. To use this option, you must specify the ``password`` argument for this request (not the ``api_token`` argument). Specifying this argument invalidates your existing API token and creates and stores a new token for future prediction requests.
  n_retry=<n_retry>             Specifies the number of times DataRobot will retry if a request fails. A value of -1, the default, specifies an infinite number of retries.
- pred_name=<pred_name>         Applies a name to the prediction column of the output file. If you do not supply the argument, the column name is blank. For binary predictions assumes last class in lexical order as positive.
+ pred_name=<pred_name>         Applies a name to the prediction column of the output file. If you do not supply the argument, the column name is blank. For binary predictions only column for positive class is put into output. Assumes last class in lexical order as positive.
```

Comment:
For binary predictions, only positive class columns are included in the output. 

This sentence still does not make sense to me. Is it "For binary predictions, only positive class columns are included in the output and the last class in lexical order is assumed positive." BUT...if ONLY positive class columns are included, then of course the last one is positive.